data-otservbr-global/scripts/actions/other/primal_bag.lua

```diff
@@ -10,7 +10,7 @@ local rewards = {
     { id = 39177, name = "spiritthorn ring" },
     { id = 39180, name = "alicorn ring" },
     { id = 39183, name = "arcanomancer sigil" },
-    { id = 38186, name = "arboreal ring" }
+    { id = 39186, name = "arboreal ring" }
```

Comment:
I think it should be 39188.

Reply:
There is 3 status:
- 39186 : brand new
- 39187 : running time (2h)
- 39188 : emty (need refill with npc)

Reply:
@Aerwix First i check the tibia wiki, and there the primal bag does not specify that it is charged, but the Fibia Fandom specifies that it is charged. My bad.